src/main/java/teammates/common/datatransfer/questions/FeedbackQuestionDetails.java

```diff
@@ -67,17 +67,19 @@ public abstract String getQuestionResultStatisticsCsv(List<FeedbackResponseAttri
     public abstract String getCsvHeader();
 
     /** Gets the header for detailed responses in csv format. Override in child classes if necessary. */
-    public String getCsvDetailedResponsesHeader() {
+    public String getCsvDetailedResponsesHeader(Boolean hasCommentsForResponses, int noOfComments) {
```

Comment:
Is there a need to use the `Boolean` wrapper class? Ideally, the primitive `boolean` variable type should suffice.